pkg/services/accesscontrol/database/resource_permissions.go

```diff
@@ -13,24 +13,28 @@ import (
 )
 
 type flatResourcePermission struct {
-	ID          int64  `xorm:"id"`
-	ResourceID  string `xorm:"resource_id"`
-	RoleName    string
-	Action      string
-	Scope       string
-	UserId      int64
-	UserLogin   string
-	UserEmail   string
-	TeamId      int64
-	TeamEmail   string
-	Team        string
-	BuiltInRole string
-	Created     time.Time
-	Updated     time.Time
+	ID            int64 `xorm:"id"`
+	RoleName      string
+	Action        string
+	Scope         string
+	ResourceScope string
```

Comment:
question: what is the difference between scope and resource scope? Is scope the scope that is used, and resource scope the scope that is based on this resources ID?

Reply:
Yes i think this name is unclear as well. Need to figure out a better one. So the scope will be what scope underlying permissions has and ResourceScope is the scope we are looking for. With those two we can determine if a permissions is inherited (for dashboards this would be folder scopes) or not